extensions/shared/external-media/editor.scss

```diff
@@ -0,0 +1,376 @@
+/**
+ * External Media
+ */
+
+$font-stack: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
+$font-size: 14px;
+$icon-size: 24px;
+$line-height: 1.6;
+$radius-roundrect: 8px;
+$radius-round: 50%;
+$grid-size: 8px;
+$tabbar-height: 48px;
+$border-width-tab: 3px;
+
+// G2 colors.
+$blue-g2: #3858e9;
```

Comment:
Maybe we can leverage `@wordpress/base-styles` and Automattic's Color studio in those stylesheets a bit more?
https://github.com/Automattic/jetpack/blob/f6f313d268c336491c8bca1afcf3e5d3663597ce/extensions/README.md#L228-L234